google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1597,11 +1599,23 @@ def iter(self):
             LOGGER.debug(e)
 
 
-class BigqueryDataSetIterator(resource_iter_class_factory(
-        api_method_name='iter_bigquery_datasets',
-        resource_name='bigquery_dataset',
-        api_method_arg_key='projectNumber')):
+class BigqueryDataSetIterator(ResourceIterator):
     """The Resource iterator implementation for Bigquery Dataset."""
+    def iter(self):
```

Comment:
The iter_bigquery_datasets isn't changing, so this doesn't need to be added explicitly.